content/docs/migration_1.0/Python.md

```diff
@@ -29,12 +29,13 @@ with session.declare_subscriber("my/keyexpr") as subscriber:
     # `subscriber.undeclare()` will be called at the end of the block`
 ```
 
-In previous version, it was necessary to keep a variable in the scope for a subscriber/queryable declared with a callback. This constraint has been lifted, and it's now possible to declare and forget an entity with a callback; in that case, the entity will keep living in background, until the session is closed. 
+In previous version, it was necessary to keep a variable in the scope for a subscriber/queryable declared with a callback. This constraint has been lifted, and it's now possible to declare a "background" entity; this entity will keep living in background, having its callback executed until the session is closed. 
 
 ```python
 import zenoh
 with zenoh.open(zenoh.Config()) as session:
-    session.declare_subscriber("my/keyepxr", lambda s: print(s))
+    # no need to declare a variable
+    session.declare_subscriber("my/keyepxr", lambda s: print(s), background=True)
     sleep(10) # subscriber stays in background and its callback can be called
     # `session.close()` will be called at the end of the block, and it will undeclare the subscriber
```

Comment:
The comment is too long, it overflows on small resolutions.
```suggestion
    # `session.close()` will be called at the end of the block,
    # and it will undeclare the subscriber
```